Tests/js/models/assets/ez-locationmodel-tests.js

```diff
@@ -905,6 +905,35 @@ YUI.add('ez-locationmodel-tests', function (Y) {
         },
     });
 
+    isRootTest = new Y.Test.Case({
+        name: "eZ Location Model is Root test",
+
+        setUp: function () {
+            this.model = new Y.eZ.Location();
+        },
+
+        tearDown: function () {
+            this.model.destroy();
+            delete this.model;
+        },
+
+        "_isLocationRoot Should return true if depth is 1": function () {
```

Comment:
ok... but in that sentence i still miss *what* should return true (the method in this case)